### PR TITLE
Setup drone ci

### PR DESCRIPTION
**Pull request by marcbachmann**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   ],
   "main": "index.js",
   "scripts": {
-    "test": "npx mocha --recursive ./test/client/document-test.js"
+    "test": "npx mocha --recursive ./test/client/client-test.js"
```
Comment (mokimo) [inline package.json]: i think the can use ./test/ incase we add more tests

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
+package-lock.json
```
Comment (disufi) [inline .gitignore]: nice catch :)

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  eslint:
+    group: install
+    image: marcbachmann/eslint:4.19.1
+    secrets: [gh_token]
+
+  test:
+    group: test
+    image: node:10-alpine
+    commands: ["npm run test -s"]
```
Comment (disufi) [inline .drone.yml]: silent, right? I think good for now 👍

File: .gitignore (hunk 2 above)
Comment (marcbachmann) [reply .gitignore]: in other projects we have `package-lock=false` in the npmrc file, but I think this is a better solution than that as developers now have the local file and can use `npm ci` to execute an install if they want that.

File: .drone.yml (hunk 3 above)
Comment (marcbachmann) [reply .drone.yml]: yes, that's the default I'm using in all the droneci configs. Silent just means that it won't print the  npm info at all. So it won't show the command that gets executed and also not the stack trace if it fails. It just shows the raw stdout and stderr of the child process.

**Pull request merged**